.github/workflows/label-pull-requests.yaml

```
@@ -1,7 +1,6 @@
 name: Label Pull Requests
 
-on:
-  - pull_request_target
+true: pull_request_target
```

actionlint (1.7.4)

unexpected key "true" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)